src/MangaJet/app/src/main/java/com/mangajet/mangajet/aboutmanga/AboutMangaVPAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package com.mangajet.mangajet.aboutmanga
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import androidx.lifecycle.Lifecycle
+import com.mangajet.mangajet.aboutmanga.aboutMangaFragment.AboutMangaFragment
+import com.mangajet.mangajet.aboutmanga.mangaChaptersFragment.MangaChaptersFragment
+
+class AboutMangaVPAdapter(fragmentManager: FragmentManager, lifeCycle : Lifecycle) :
+    FragmentStateAdapter(fragmentManager, lifeCycle) {
+    override fun getItemCount(): Int {
+        return 1 + 1
```

Comment:
Make a constant. And it better be constant in AboutMangaActivity.kt.
So you can access to it like
AboutMangaActivity.TABS_COUNT